<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddScooterDialogClass</class>
 <widget class="QDialog" name="AddScooterDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddScooterDialog</string>
  </property>
  <widget class="QDoubleSpinBox" name="pricePerDaySpinbox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>200.000000000000000</double>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_5">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="plainText">
    <string>price_per_day</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="pricePerHourSpinbox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.100000000000000</double>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>15.000000000000000</double>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="manufacturerEditText">
   <property name="geometry">
    <rect>
     <x>123</x>
     <y>20</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="placeholderText">
    <string>manufacturer</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>360</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="productionYearEditText">
   <property name="geometry">
    <rect>
     <x>273</x>
     <y>100</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="placeholderText">
    <string notr="true">1999</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="modelEditText">
   <property name="geometry">
    <rect>
     <x>183</x>
     <y>60</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="placeholderText">
    <string>model</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="plainText">
    <string>price_per_hour</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
